### PR TITLE
Upgrade to kubernetes.model.version 1.0.66

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

**Comment by rawlingsj** [comment]
This PR includes the upgrade of kubernetes-model to using openshift and kubernetes 1.5.x

**Comment by rawlingsj** [comment]
@iocanel @jimmidyson I've rebased this PR, would you be able to review when you get 2 mins and we can release it, I'll then take care of fabric8, fabric8-maven-plugin and downstream projects.  Thanks

**Comment by jimmidyson** [comment]
@iocanel Any concerns with this before I merge?

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/ExtensionsAPIGroupClient.java [hunk 1]
```diff
@@ -23,15 +23,15 @@
...
+import io.fabric8.kubernetes.api.model.DoneableJob;
 import io.fabric8.kubernetes.api.model.extensions.DoneableReplicaSet;
 import io.fabric8.kubernetes.api.model.extensions.DoneableThirdPartyResource;
 import io.fabric8.kubernetes.api.model.extensions.HorizontalPodAutoscaler;
 import io.fabric8.kubernetes.api.model.extensions.HorizontalPodAutoscalerList;
 import io.fabric8.kubernetes.api.model.extensions.Ingress;
 import io.fabric8.kubernetes.api.model.extensions.IngressList;
-import io.fabric8.kubernetes.api.model.extensions.Job;
-import io.fabric8.kubernetes.api.model.extensions.JobList;
+import io.fabric8.kubernetes.api.model.Job;
```
Comment (jstrachan) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/ExtensionsAPIGroupClient.java]: how about we go for version 2?

Comment (rawlingsj) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/ExtensionsAPIGroupClient.java]: version 2 sounds good to me

**Comment by iocanel** [comment]
@jimmidyson: No concerns from my side.

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/ExtensionsAPIGroupClient.java (hunk 1 above)
Comment (jimmidyson) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/ExtensionsAPIGroupClient.java]: Done - can you bump version to 2.0-SNAPSHOT then please

Comment (rawlingsj) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/ExtensionsAPIGroupClient.java]: Sure thing

**Comment by jimmidyson** [comment]
Thanks @rawlingsj - [merge]

**Pull request merged**